Documentation/security/policy/language.rst

```diff
@@ -719,11 +721,15 @@ which is defined as follows:
 
         // PortProtocol specifies an L4 port with an optional transport protocol
         type PortProtocol struct {
-                // Port is an L4 port number. For now the string will be strictly
-                // parsed as a single uint16. In the future, this field may support
-                // ranges in the form "1024-2048"
+                // Port can be an L4 port number, or a name in the form of "http"
+                // or "http-8080".
                 Port string `json:"port"`
 
+                // EndPort can only be an L4 port number.
+                //
+                // +optional
+                EndPort string `json:"endPort,omitempty"`
```

Comment:
Hmm. The type here is incorrect:

https://github.com/cilium/cilium/commit/4fe60e5214347

Reply:
^ Still inconsistent, the actual API is int32

Reply:
:facepalm: